accounts-db/src/accounts_db/geyser_plugin_utils.rs

```diff
@@ -121,7 +121,7 @@ impl AccountsDb {
 
             // later entries in the same slot are more recent and override earlier accounts for the same pubkey
             // We can pass an incrementing number here for write_version in the future, if the storage does not have a write_version.
-            // As long as all accounts for this slot are in 1 append vec that can be itereated olest to newest.
+            // As long as all accounts for this slot are in 1 append vec that can be iterated olest to newest.
```

Comment:
nit: Since you're already in here, wanna fix "olest" to "oldest"

Reply:
fixed! good eyes lol